packages/odata-v2/src/uri-conversion/odata-uri.spec.ts

```diff
@@ -64,7 +71,7 @@ describe('getFilter', () => {
     );
   });
 
-  it('for nested filters with and operator only', () => {
+  it("for nested filters with 'and' operator only", () => {
```

Comment:
tI know it was the way before, but for me it is not clear what it does. I prefer a verb like 'transforms' or so.

Reply:
I agree, done.